Fix ImaAdsLoader handling of first midroll ad without preroll #5928

foodpoison opened the pull request:
In situation where we have a IMA tag setup of no preroll but have midrolls,
and when we start playback directly after first midroll, we can get into a
stuck state after ad finishes playing. This patch fixes this issue.

For more details about the issue see #5928

AquilesCanta commented:
@andrewlewis mind taking a look?

foodpoison commented:
Hi there, any update on this, please?

andrewlewis commented:
@foodpoison Sorry for the delay. This looks good. Please could you send it to the dev-v2 branch and we'll merge it?

foodpoison committed:
Fix ImaAdsLoader handling of first midroll ad without preroll

In situation where we have a IMA tag setup of no preroll but have midrolls,
and when we start playback directly after first midroll, we can get into a
stuck state after ad finishes playing. This patch fixes this issue.

andrewlewis commented:
@foodpoison We have an equivalent internal change so will push that. The issue will be updated once that is done. Thanks for the PR!
